schema/schema.go

```diff
@@ -280,7 +280,9 @@ func GenerateFromField(f reflect.StructField, mode Mode) (string, bool, *Schema,
 		if err != nil {
 			return name, false, nil, err
 		}
-		s.ExclusiveMinimum = &min
+		s.Minimum = &min
+		t := true
```

Comment:
nit: should these use a constant?

Reply:
Is there ever a condition under which they could be false? Or is it the case that if they're present, they're true? This isn't to say that they should ever be false; from the logic, if they're false, they can be safely omitted, so their presence kinda _implies_ that they're true, but that doesn't appear guaranteed from the spec.

Reply:
@lgarrett-isp I left these here to not pollute the global namespace with a constant that just means true. Unfortunately in Go you can't take the address of a primitive (i.e. can't do `foo := &(true)`) so this forces you to make two statements. If we make this a global what do we name it?

@ktracer-isp the value is false (and omitted when marshalled) by default. If the `exclusiveMinimum` field tag is present in the Go code we will **always** set this to `true` (assuming the parsing of the tag's value succeeds above). It's just on two lines because of Go limitations.

Reply:
That sounds good to me.

I was thinking about memory but under no circumstance should we be optimizing to this level (especially for _boolean_ primitives!): I was curious if this allocates two heap values (or for a whole schema, 2n where n is the number of fields). It was a curiosity and I definitely don't think we should change it for that reason--and as you note, a constant is also an awkward mental load I think (`DEFAULT_TRUE = true`?) so either approach seems equally annoying and I'm ready to leave it as is :+1:.

Reply:
I suppose my question is whether we ever expect someone to try to explicitly set it to false, which would break. I mean, we don't have to allow this case, but if it happened this would be an instance of this code consciously choosing to do the "wrong thing," which has a smell to me. I totally agree that users should never do this, but if we're not doing anything to prevent it, we're still setting folks up for a bad time.

Reply:
If they try to set it to false in the struct tag it will fail to parse as a number, so we should be covered there honestly. Nice call out though!

Reply:
Ah! I missed that bit; that's totally legit! Good point!